shared/actions/__tests__/provision.test.tsx

```diff
@@ -270,9 +270,9 @@ describe('device name error path', () => {
 })
 
 describe('other device happy path', () => {
-  const mobile = {deviceID: '0', name: 'mobile', type: 'mobile'} as any
-  const desktop = {deviceID: '1', name: 'desktop', type: 'desktop'} as any
-  const backup = {deviceID: '2', name: 'backup', type: 'backup'} as any
+  const mobile = {deviceID: '0', name: 'mobile', type: RPCTypes.DeviceTypeV2.mobile} as any
```

Comment:
You might be able to get rid of these `as any` now. Not sure why they're there. But also it's ok if you don't want to do any more fighting with `tsc` on this pr.

Reply:
I tried to start typing `tsc` as well, but things were snowballing out of the 1 point for this ticket, so I decided to stop at the "tsc border".